lib/ipc/fs.c

```diff
@@ -0,0 +1,567 @@
+#include<libtransistor/types.h>
```

Comment:
add a comment that this file is autogenerated and should probably not be modified